### PR TITLE
Using plugin in a receive-only mode w/o connection string

### LINKED ISSUE #24: Using plugin in a "receive only" mode w/o connection string
Hi - great work.  Appologies if I am missing something obvious - I have a working sample of a sender and reciever, but it is not quite clear to me how you configure the receiver such that it does not need the storage account connection string - I think I see in the code of the plugin itself that it may not leverage the connection string in the case that the message has the SAS uri, but the connection string still seems to be required in order to register the plugin with the MessageReceiver (connection string is a required parameter to AzureStorageAttachmentConfiguration constructor, and it looks like the plugin's constructor would blow if it weren't available, as it attaches to the account at that time).  Thanks!

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #24

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBus.AttachmentPlugin 0.0.131-preview completed](https://ci.appveyor.com/project/seanfeldman/servicebus-attachmentplugin/build/0.0.131-preview) (commit https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin/commit/219fcc3fec by @SeanFeldman)

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBus.AttachmentPlugin 0.0.134-preview completed](https://ci.appveyor.com/project/seanfeldman/servicebus-attachmentplugin/build/0.0.134-preview) (commit https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin/commit/b5246eb58e by @SeanFeldman)

**Comment by SeanFeldman** [comment]
@bledbet is [this](https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin/pull/27/commits/7eb7801f266e84ea796be9338860c84cb07718b7) aligned with what you thought of?

File: src/ServiceBus.AttachmentPlugin.Tests/When_using_receive_only_plugin.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        public async Task Should_download_attachment_using_provided_from_sas_uri()
+        {
+            var payload = "payload";
+            var bytes = Encoding.UTF8.GetBytes(payload);
+            var message = new Message(bytes)
+            {
+                MessageId = Guid.NewGuid().ToString(),
+            };
+            var plugin = new AzureStorageAttachment(new AzureStorageAttachmentConfiguration(
+                    connectionString: "UseDevelopmentStorage=true", containerName: "attachments", messagePropertyToIdentifyAttachmentBlob: "attachment-id")
```
Comment (SimonCropp) [inline src/ServiceBus.AttachmentPlugin.Tests/When_using_receive_only_plugin.cs]: minor, but i would prefer newlines for each param  to improve readability

**Comment by SeanFeldman** [comment]
:x: [Build ServiceBus.AttachmentPlugin 0.0.136-preview failed](https://ci.appveyor.com/project/seanfeldman/servicebus-attachmentplugin/build/0.0.136-preview) (commit https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin/commit/c0896f8eb8 by @SeanFeldman)

**Comment by SeanFeldman** [comment]
:x: [Build ServiceBus.AttachmentPlugin 0.0.137-preview failed](https://ci.appveyor.com/project/seanfeldman/servicebus-attachmentplugin/build/0.0.137-preview) (commit https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin/commit/c0896f8eb8 by @SeanFeldman)

**Pull request merged**